lib/server/push.api.js

```diff
@@ -24,6 +24,14 @@ Push.setBadge = function (/* id, count */) {
   // throw new Error('Push.setBadge not implemented on the server');
 };
 
+Push.getBadgeNumber = function() {
```

Comment:
@IlicPavle @mjovkovic Because this function exists on `Push` object which is exported, what's the purpose of `Push.setBadgeFunction` since we can just write 
```
Push.getBadgeNumber = () => {
  // some code
}
```

Previously, we had `PushOverrides`, which had only "setter" functions, and `getBadgeNumber` function vas global to this package, and not visible outside. Keep in mind that we will need more "setter" functions for other parts as well (to remove `GlobalVets`).